attestation/commandrun/commandrun.go

```diff
@@ -160,6 +160,10 @@ func (rc *CommandRun) RunType() attestation.RunType {
 	return RunType
 }
 
+func (rc *CommandRun) EnableTracing() bool {
```

Comment:
This may be nitpicky, but could we rename this `TracingEnabled` instead? `EnableTracing` reads to me like "take the action to enable tracing" versus "ask if tracing is enabled".

Reply:
done!